processor/intervalprocessor/processor.go

```diff
@@ -65,8 +65,8 @@ func newProcessor(config *Config, log *zap.Logger, nextConsumer consumer.Metrics
 		summaryLookup:      map[identity.Stream]pmetric.SummaryDataPoint{},
 
 		exportInterval:     config.Interval,
-		gaugePassThrough:   config.GaugePassThrough,
-		summaryPassThrough: config.SummaryPassThrough,
+		gaugePassThrough:   config.PassThrough.Gauge,
+		summaryPassThrough: config.PassThrough.Summary,
```

Comment:
how about just storing the entire `config` struct in `processor`, instead of manually copying literally all fields?

Reply:
Good idea